packages/foam-vscode/src/services/editor.ts

```diff
@@ -175,8 +175,11 @@ export const createMatcherAndDataStore = async (
     return files.map(fromVsCodeUri);
   };
 
-  const readFile = async (uri: URI) =>
-    (await workspace.fs.readFile(toVsCodeUri(uri))).toString();
+  const decoder = new TextDecoder('utf-8');
```

Comment:
I am planning to add this to master, so that this PR is focused on configuration changes required for the switch to web extension
(I consider the current implementation of this fn to be a bug that happens to work by chance)